components/shared_server_code/tests/shared/database/test_measurements.py

```diff
@@ -1,30 +1,25 @@
 """Unit tests for the measurements collection."""
 
-import unittest
 from unittest.mock import Mock
 
-from shared_data_model import DATA_MODEL
-
 from shared.database.measurements import insert_new_measurement, latest_measurement, recent_measurements
 from shared.model.measurement import Measurement
 from shared.model.metric import Metric
 
 from tests.fixtures import METRIC_ID, METRIC_ID2
 
+from ..base import DataModelTestCase
 
-class MeasurementsTest(unittest.TestCase):
-    """Unit test for getting and inserting measurements."""
 
-    @classmethod
-    def setUpClass(cls) -> None:
```

Comment:
Is it ok to change from setUpClass to setUp? We weren't using any of the implicit setUpClass aspects?

Reply:
Yes, this works. First `DataModelTestCase.setUpClass()` is invoked once to create the `DATA_MODEL` class attribute and then `MeasurementsTest.setUp()` is called before each unit test to set up the database fixture.